research/src/pages/component-spec-template.mdx

```diff
@@ -0,0 +1,213 @@
+
+*Example Table*
+
+| State Group | States | Initial State | Description | Interaction/Transition |
+| ----------- | ------ | ------------- | ----------- | ---------------------- |
+| `disabled` | `true`/`false` | `false` | When `true`, disables the control, preventing user interaction and displaying the control with a disabled appearance. | No interaction. Controlled programmatically. |
+
+### Accessibility
+
+*Consider the accessibility of the component.*
+
+#### Keyboard Navigation and Focus
+
+- *Arrow vs tabbing key behavior*
+- *Modifier key effects (e.g. shift, ctrl, meta)*
+- *Does this component use focus delegation?*
+
+*Example Table*
+
+| Key | Description |
+| --- | ----------- |
+| Up/Right  | Increments the value of the component by calling the `increment` method. If the `shift` modifier is pressed, increases by 10x the `step` value. |
+| Down/Left | Decrements the value of the component by calling the `decrement` method. If the `shift` modifier is pressed, decreases by 10x the `step` value. |
+
+#### Form Input
+
+*Does this component integrate with form submission, form validation, etc.? Can integration be accomplish with standard APIs or are special work-arounds required?*
+
+#### Use with Assistive Technology
+
+*Are there any details about shadow dom, focus delegation, or aria attributes that need special attention?*
+
+### Globalization
+
+*Consider whether the component has any special globalization needs such as:*
+
+- *Special RTL handling*
+- *Swapping of internal icons/visuals*
+- *Localization*
+
+### Security
+
+*Are there any security implications surrounding the component?*
+
+### Performance
+
+*Are there any performance pitfalls or challenges with implementing the component? (examples below)*
+
+* If the component renders a loop of items, should certain areas be considered for virtualization?
+* If the component needs to measure an area before rendering, how will jank be avoided?
+* If any measuring is needed at all, can rAF techniques help queue measures and prevent synchronous reflows?
+
+### Dependencies
+
+*Will implementing the component require taking on any dependencies?*
+
+- *3rd party libraries*
+- *Upcoming standards we need to polyfill*
+- *Do any dependencies bring along an associated timeline?*
+
+#### Platform Requirements
+
+*Are there any core web platform features that are needed to implement this component well?*
+
+### Test Plan
+
+*Are there any additional details that need to be tested which aren't covered in the spec above? Are there any expected nuances or challenges with testing this component?*
+
+### Tooling
+
+*Are there any special considerations for tooling? Will tooling changes need to be made? Is there a special way to light up this component in dev tools that would be compelling for developers/designers?*
+
+---
+
+## Resources
+
+*Any related resource links such as web standards, discussion threads, diagrams, etc.*
+
+---
+
+## Next Steps
```

Comment:
I would expect these to be in Github issues. Possibly we should have a generic statement but link to the label for this specific specification so that the user can filter on enhancements, issues, etc. Something like:

> Open UI track all feature additions, modifications, bugs or even editorial changes in our Github repo. Click here to review this specification's issues.

Reply:
Yes, this could be a list of links to follow-on issues or also just thoughts on what could be done with the component beyond the initial MVP outlined by the spec.

Reply:
I added some clarifying words based on your recommendation.